e2e/react-start/dev-ssr-styles/src/styles/css-import-base.css

```
@@ -0,0 +1,5 @@
+.css-import-order {
+  --css-import-base-marker: issue-7794;
+  background-color: #ffffff;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)